regexes/device/mobiles.yml

```diff
@@ -1407,17 +1407,11 @@ Blaupunkt:
 
 # Bmobile
 Bmobile:
-  regex: 'Bmobile[_ ]|AX10(?:[1234567]0|[1345678]5|16|65E|70e|73\+|7[1234]|91)|AX(?:5[12347]0|512|5[123]5|524|6[01256789]0|[67]05|675|68[0135]|680\+|710|745|810|82[013]|92[012])|AX-340|AX7OO'
+  regex: 'Bmobile[_ ]|AX-?([1-9][0-9eEO+]{2,4})'
```

Comment:
I guess the `eEO+` can only be at the end, right? So maybe we should use `AX-?([1-9][0-9]{2,3}[eEO+])`

Reply:
You forgot to add a `?` sign. Also, they made a typo mistake on the user agent, instead of 700 they wrote 7OO, but on their website the model is 700. I think this is the final regex `AX-?([1-9][0-9]{2,3}[eEO+]?|7OO)`